frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -286,6 +286,12 @@ function ReportStreamNavbar({
                 Support
             </USSmartLink>
         </div>,
+        <div className="primary-nav-link-container" key="support">
```

Comment:
What are these used for?

Reply:
fake links to trigger the bad link check to produce output, they will be removed after

Reply:
Just need to remove these and LGTM